wp-web-push/lib/js/wp-web-push.js

```diff
@@ -38,16 +38,42 @@ if (navigator.serviceWorker) {
     }, 200);
   }
 
+  function showSubscribe() {
+    document.querySelector('#webpush-subscription .message').innerHTML = '<p>' + ServiceWorker.subscription_prompt + '</p><p><img src="' + ServiceWorker.notification_preview + '" alt="" /></p>';
+    var actionButton = document.querySelector('#webpush-subscription .actions .default');
+    actionButton.innerHTML = ServiceWorker.subscription_button_text;
+    actionButton.onclick = function () {
+      enableNotifications()
```

Comment:
Use `enableNotifications(true)` here (`true` means that we ignore the interval between prompts, which we should do if the user explicitly clicks on the button).